.gitignore

```diff
@@ -360,3 +360,7 @@ MigrationBackup/
 # venv
 env/
 website/docusaurus/
+
+tools/awps-tunnel/client/public/examples/
+
+tools/awps-tunnel/client/src/components/api/restapiSample.json
```

Comment:
possible to have these 2 in the same folder as similar to how the rest api github organize the files?